src/openrct2/GameStateSnapshots.cpp

```diff
@@ -233,7 +233,7 @@ struct GameStateSnapshots : public IGameStateSnapshots
         COMPARE_FIELD(Peep, nausea_tolerance);
         COMPARE_FIELD(Peep, window_invalidate_flags);
         COMPARE_FIELD(Peep, paid_on_drink);
-        for (int i = 0; i < PEEP_MAX_THOUGHTS; i++)
+        for (int i = 0; i < 16; i++)
```

Comment:
Why not use a constant or `std::size`? (I know `PEEP_MAX_THOUGHTS` was the wrong constant.)

Reply:
I'm also wondering this.

Reply:
probably because there isn't a constant for it right now.

Reply:
There is still std::size.